prep-html-syntax/index.html

```diff
@@ -0,0 +1,3 @@
+<h1 style="color:crimson;">My favorite Star Wars character is Rey!</h1>
```

Comment:
Hey Angel!! You are missing your HTML Skeleton here, and that is a mandatory part of any HTML document. Please add your HTML skeleton and place this content in the `body` element.

Reply:
Oops! I didn't realize we were building on the previous assignment. I just made a new a commit and pushed the changes! :)